epub33/rs/index.html

```diff
@@ -1072,14 +1072,14 @@ <h4>The <code>rendition:page-spread-*</code> Properties</h4>
 							regular and center-spread pages.</p>
 					</div>
 
-					<p id="page-layout-both">When a <a href="#def-layout-reflowable">reflowable</a> spine item follows a
+					<p id="page-layout-both" data-tests="#page-layout-both">When a <a href="#def-layout-reflowable">reflowable</a> spine item follows a
 							<a href="#def-layout-pre-paginated">pre-paginated</a> one, the reflowable one SHOULD start
 						on the next page &#8212; as defined by the <a
 							href="https://www.w3.org/TR/epub-33/#attrdef-spine-page-progression-direction"
 								><code>page-progression-direction</code></a> [[EPUB-33]] &#8212; when it lacks a
 							<code>rendition:page-spread-*</code> property value.</p>
```

Comment:
Good news: I finally understand this after reading it a zillion times. But I think I would have understood it much quicker if this paragraph came after the #page-spread-flow paragraph below (and the paragraph that follows). @mattgarrish can we move line 1075 down to line 1090 (in a separate PR, with a mild rewrite to make it make sense)?